src/components/TextField/TextField.tsx

```diff
@@ -7,6 +7,7 @@ type Props = {
   label?: string,
   required?: boolean,
   onChange?: (newValue: string) => void,
+  isValid?: (url: string) => boolean;
```

Comment:
```suggestion
  validateValue?: (value: string) => boolean;
```

Name it as a function. Also, it can check any value, not only urls